docs/tools/fbsql/fbsql-home.md

```diff
@@ -176,6 +176,16 @@ expanded (or x)
 
 If **value** is specified it must be either on or off, which will enable or disable expanded mode, or auto. If value is omitted the command toggles between the on and off settings. When expanded mode is enabled, query results are displayed in two columns, with the column name on the left and the data on the right. This mode is useful if the data wouldn't fit on the screen in the normal “horizontal” mode.
```

Comment:
Ok so this is the old standard for writing content and not the new one.
I'm going to let this pass because the fbsql-home file needs to be revised to meet the new standards.

If you want an example of how to structure information like this, look to the sql-guide/statements files which give a clear explanation of syntax, a table of arguments followed by additional information which is more detailed (where stuff like this would go).